geoportailv3/static/js/layerinfo/popup.html

```diff
@@ -0,0 +1,5 @@
+<h4 class="popover-title ngeo-popup-title">
+  <span>{{title | translate}}</span>
+  <button type="button" class="close" ng-click="open = false"> &times;</button>
+</h4>
+<div class="popover-content" ng-bind-html="content"></div>
```

Comment:
Do we need to translate the content? Or does the metadata service return already-translated responses? @jaykayone, any idea?